.tito/custom.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2016 Red Hat, Inc.
+#
+# All rights reserved.
+#
+# Author: Jan Pokorny <jpokorny@redhat.com>
+# (using src/tito/builder/fetch.py from the tito project as a template)
+#
+# This source code is licensed under the GNU Lesser General Public License
+# version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+#
+
```

Comment:
docstrings (PEP-0257) are definitely more pythonic than comments for the top matter.

The particular names of double-underbar metadata variables are nonstandard, but are a common convention mentioned in PEP-8 ("dunders"). It can make sense to document the author when different from copyright holder, and we can update the style guide to explicitly mention that (it already implicitly lets the docstring contain such information), though personally I think the git history is a better source for authorship (copyright and license are obviously important in the file itself for legal reasons).

We can update the style guide to parametrize the copyright holder, and maybe add an example with everything filled in so it's clearer.

More than particular style choices, maintaining consistency across the project makes it easier for a new contributor to learn and follow the code.